### DISCUSSION TITLE
Separate CPG-Languages and persistence into modules that can be compiled selectively

### DISCUSSION THREAD

**Discussion by konradweiss** [Fraunhofer-AISEC/cpg#415] [Ideas]
Separate functions and language support into modules to allow testing of functionality together while allowing separate publishing and importing.
- cpg-core
- cpg-python
- cpg-java
- cpg-cpp

- cpg-neo4j
- cpg-overflow

**Comment by oxisto** [comment]
Initial support for modules addressed in #417
